bundles/certified-operators/4.4.19/manifests/minio-operator.clusterserviceversion.yaml

```diff
@@ -489,7 +489,7 @@ spec:
                                 - minio-operator
                         topologyKey: kubernetes.io/hostname
                 containers:
-                  - image: quay.io/minio/operator:v4.4.18
+                  - image: quay.io/minio/operator:4.4.19
```

Comment:
There is no such thing as 

```
docker pull quay.io/minio/operator:4.4.19
Error response from daemon: manifest for quay.io/minio/operator:4.4.19 not found: manifest unknown: manifest unknown
```

Reply:
But there will be after you push it :P @harshavardhana

Reply:
no 

```
docker pull quay.io/minio/operator:4.4.18
Error response from daemon: manifest for quay.io/minio/operator:4.4.18 not found: manifest unknown: manifest unknown
```

Reply:
```sh
$ docker pull quay.io/minio/operator:v4.4.18
v4.4.18: Pulling from minio/operator
Digest: sha256:9d9e05924e7d9faf936f71f325559066ef146f6b3360e43b608f57f8d5611a3c
Status: Image is up to date for quay.io/minio/operator:v4.4.18
quay.io/minio/operator:v4.4.18
```